frontend/src/components/App.js

```diff
@@ -1,26 +1,79 @@
 import React from 'react';
 import EventList from "./EventList";
 import './App.css';
+import Grid from "@material-ui/core/Grid";
 
-function App() {
-    let events = [
-        {
-            "description": "Room 312",
-            "start": "2020-07-25",
-            "end": "2020-07-26",
-        },
-        {
-            "description": "Room 311",
-            "start": "2020-07-27",
-            "end": "2020-07-29",
+import Paper from "@material-ui/core/Paper";
+import DatePicker from "./DatePicker";
+
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            events: [],
+            start: new Date()
+        }
+    }
+
+    componentDidMount() {
+        fetch("http://localhost:5000/api/calendar/", {
+            method: 'POST',
+            body: JSON.stringify(this.state.start)
```

Comment:
you backend implementation expects an object with start property but you send a string